tests/integration/bootstrap-get-images.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+# This script generates "get-images.sh" using Official Images tooling.
+#
+#   ./bootstrap-get-images.sh > get-images.sh
+#
+# This script requires "bashbrew". To get the latest version, visit
+# https://github.com/docker-library/bashbrew/releases
+
+images=(
+	# pinned to an older BusyBox (prior to 1.36 becoming "latest") because 1.36.0 has some unresolved bugs, especially around sha256sum
+	'https://github.com/docker-library/official-images/raw/eaed422a86b43c885a0f980d48f4bbf346086a4a/library/busybox:glibc'
+
+	# pinned to an older Debian Buster which has more architectures than the latest does (Buster transitioned from the Debian Security Team to the LTS Team which supports a smaller set)
+	'https://github.com/docker-library/official-images/raw/ce10f6b60289c0c0b5de6f785528b8725f225a58/library/debian:buster-slim'
+)
+
+cat <<'EOH'
+#!/bin/bash
+
+# DO NOT EDIT!  Generated by "bootstrap-get-images.sh"
```

Comment:
Maybe CI should validate that the file is not modified?

Reply:
The whole point of having this bootstrap script is that we do not need extra dependencies in CI (bashbrew in the current implementation).

OTOH it can be a separate small job, let me take a look.

Reply:
OK I've added a validation job and tested that it succeeds and fails when it is supposed to.

@tianon can you please pick up https://github.com/opencontainers/runc/pull/3709/commits/38c38a98adb433373a19d8a734e128d7d0f0132a  to this PR?

Reply:
Done :+1:

Reply:
awesome!

@AkihiroSuda test added, PTAL